web/app/js/components/Navigation.test.jsx

```diff
@@ -167,11 +183,15 @@ describe('Namespace Select Button', () => {
       </BrowserRouter>
     );
 
-    const button = component.find("Button");
+    const button = component.find("button");
     expect(button).toIncludeText("emojivoto");
   });
 
   it('opens the Namespace Selection menu if button is clicked', () => {
+    // Material UI - Popover needs a proper anchorEl if NOVE_ENV is different to "test"
```

Comment:
Mind changing `NOVE_ENV` to `NODE_ENV`?

I would change the whole comment to be: 
```
// Material-UI v4 requires that the popover needs a valid anchorEl if NODE_ENV is not "test"
```